packages/cli/lib/prompts/enterAccountNamePrompt.js

```diff
@@ -27,7 +31,53 @@ const enterAccountNamePrompt = defaultName => {
   return promptUser(accountNamePrompt(defaultName));
 };
 
+// Used for HubSpot portal account names
+const hubspotAccountNamePrompt = ({ accountType, currentPortalCount = 0 }) => {
+  const isDevelopmentSandbox =
+    accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX;
+  const isSandbox =
+    accountType === HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX ||
+    isDevelopmentSandbox;
+  const isDeveloperTestAccount =
+    accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPER_TEST;
+
+  let promptMessageString;
+  let defaultName;
+  if (isSandbox) {
+    promptMessageString = isDevelopmentSandbox
+      ? i18n(`${i18nKey}.enterDevelopmentSandboxName`)
+      : i18n(`${i18nKey}.enterStandardSandboxName`);
+    defaultName = i18n(`${i18nKey}.sandboxDefaultName`, {
+      sandboxType: isDevelopmentSandbox ? 'development' : 'standard',
+    });
+  } else if (isDeveloperTestAccount) {
+    promptMessageString = i18n(`${i18nKey}.enterDeveloperTestAccountName`);
+    defaultName = i18n(`${i18nKey}.developerTestAccountDefaultName`, {
+      count: currentPortalCount + 1,
+    });
+  }
+
+  return promptUser([
+    {
+      name: 'name',
+      message: promptMessageString,
+      validate(val) {
+        if (typeof val !== 'string') {
+          return i18n(`${i18nKey}.errors.invalidName`);
+        } else if (!val.length) {
+          return i18n(`${i18nKey}.errors.nameRequired`);
+        }
+        return accountNameExistsInConfig(val)
+          ? i18n(`${i18nKey}.errors.accountNameExists`, { name: val })
+          : true;
+      },
+      default: defaultName,
+    },
+  ]);
+};
+
 module.exports = {
   accountNamePrompt,
   enterAccountNamePrompt,
```

Comment:
Sorry - thought of this after already approving but it i feel like it make make sense to rename these existing methods to something like `cliAccountNamePrompt` to clarify how they're different from `hubspotAccountNamePrompt`

Reply:
I can handle this since this was already merged! My bad for not thinking of it sooner

Reply:
Agh merged too soon, but yes that would def clarify things. was going to say I can PR it but I can leave it to you if you'd like!

Reply:
yeah np i got it!